source/_integrations/shelly.markdown

```diff
@@ -199,6 +200,21 @@ The firmware limits the transition time to 5 seconds.
 
 </div>
 
+## Device services
+
+The integration offers device services, which can be triggered by a configuration button.
+
+### OTA update
+
+It is possible to trigger a so called OTA firmware update of your devices.
```

Comment:
```suggestion
Trigger device OTA firmware update.
```
I am not sure if my wording is ok, but I think documentation should not be expressed as "so called"